.github/workflows/build-and-push-docker.yml

```diff
@@ -57,6 +57,11 @@ on:
         default: false
```

Comment:
This file was first commit, so would need to be reviewed fully.

Setup is taken from `frontend-mono`